fs_brokers/apache_hdfs_broker/src/main/java/org/apache/doris/broker/hdfs/FileSystemManager.java

```diff
@@ -419,7 +420,7 @@ public BrokerFileSystem getS3AFileSystem(String path, Map<String, String> proper
                 conf.set(FS_S3A_ACCESS_KEY, accessKey);
                 conf.set(FS_S3A_SECRET_KEY, secretKey);
                 conf.set(FS_S3A_ENDPOINT, endpoint);
-                conf.set(FS_S3A_IMPL_DISABLE_CACHE, "true");
+                conf.set(FS_S3A_IMPL_DISABLE_CACHE, disableCache);
```

Comment:
Under what circumstances do we need enable this cache?

Reply:
under multi process circumstances, we need set the parameter to false, avoid delete dir of other user process because the name is same conflict. for a few ago, we just encounter the circumstances. so i commit the pr, but default true to user is ok.
https://blog.csdn.net/jyl1798/article/details/42521789 is a factor for the pr @morningman